tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -151,6 +151,15 @@ sub push_junit_log {
     parse_junit_log("/tmp/output.xml");
 }
 
+sub upload_supportconfig_log {
+    my $datetab = script_output("date '+%Y%m%d%H%M%S'");
+    script_run("supportconfig -t /var/log -B supportconfig.$datetab", 600);
+    script_run("tar zcvfP /var/log/nts_supportconfig.$datetab.tar.gz /var/log/nts_supportconfig.$datetab");
+    upload_logs("/var/log/nts_supportconfig.$datetab.tar.gz");
+    script_run("rm -rf /var/log/nts_supportconfig.*");
+    save_screenshot;
+}
+
```

Comment:
Please move this func to virt_utils.pm, which is the place for such utility functions.

Reply:
Will change code as your mentioned.